### ISSUE TITLE
doc update

### ISSUE THREAD

**Issue by yuuuxt** [Arshia001/FSharp.GrpcCodeGenerator#18]
Hi, I'm learning to use this project, so please correct me if I'm wrong: 
Should [this example](https://github.com/Arshia001/FSharp.GrpcCodeGenerator/blob/40e9bce83f13e4d3cae51af1e5e27c9dd3442ce0/README.md?plain=1#L193) be updated according to [this commit](https://github.com/Arshia001/FSharp.GrpcCodeGenerator/commit/f3c921196684e55a2edb31a581a28df8f15074b2)?

**Comment by Arshia001** [comment]
Yes it should. Thanks for reporting it. I'll read the README and update old code.
